internal/configuration/env_config.go

```diff
@@ -53,41 +54,42 @@ func init() {
 	settings["GetDebugMode"] = Setting{"JC_DEBUG_MODE", defaultDebugMode, []func(interface{}, string) error{util.IsBool}}
 }
 
+// Setting is an element in the proxy configuration. Many of such Setting constitutes full configuration.
 type Setting struct {
 	key          string
 	defaultValue string
 	validations  []func(interface{}, string) error
 }
 
-// EnvConfig reads the configuration from the environment
+// EnvConfig reads the configuration from the environment.
```

Comment:
"EnvConfig is a Configuration implementation which reads the configuration from the process environment."